src/core/Network.ts

```diff
@@ -15,7 +15,7 @@
  */
 
 import ripemd160 = require('ripemd160');
-import { Keccak, SHA3 } from 'sha3';
+import { Hash } from 'js-sha3';
```

Comment:
I would recommend having our own hasher class/util/interface with our own similar API so if we need to change the implementation, the 'js-sha3' import is only in one file.